tests/test-definition.yaml

```diff
@@ -1,7 +1,7 @@
 # These tests can run against an OpenShift cluster, provided you note the following:
 #
 # 1. Set the "openshift" dimension below to "true" (with quotes)
-# 2. Comment out the "true" option in the "s3-use-tls" dimension
+# 2. Comment out the "true" option in the "s3-use-tls" dimension TODO: Trino requires S3 to use TLS.
```

Comment:
@sbernauer, shall I just remove the whole comment block about OpenShift?

Reply:
I'm not sure. I understand the note line 6-10 in such a way, that we only have MinIO without TLS on OS at the moment. This would imply we can not run any smoke tests on openshift any more.

On the other hand line 142 indicates that OS MinIO requires TLS :thinking: 
I think the answer can only be found by trying it on replicated...

Reply:
> This would imply we can not run any smoke tests on openshift any more.

Correct. The Native S3 implementation requires TLS. So I have removed the TLS toggle from tests. That means (assuming the notes above still apply) that it cannot be tested on OpenShift.

Reply:
> I think the answer can only be found by trying it on replicated...

Ok, so fast-forward, if it still doesn't work on OpenShift, what are our options?
I imagine we won't pull this feature just because of that, so we either need to:
- Accept that testing won't work on OpenShift, or
- Spend time fixing tests (which I'd rather put into the planning process than deal with it now - as this task has already balooned).

Reply:
I'm with you here. Let's accept the fact for now. I guess you can then reduce the notes to "Does not work on OS, because <existing explanation>"

Reply:
I have updated the whole block: f42a9013bd356dea37375df2ac61e18c91a584a2

We can test it next time it comes up.

Reply:
Thanks!

Reply:
And 837368e8047706e3efb0057afcaa6815bb7a6f6e